README.md

```diff
@@ -353,7 +353,7 @@ Copyright &copy; 2021 [Mineiros GmbH][homepage]
 [badge-build]: https://github.com/mineiros-io/terraform-aws-iam-role/workflows/CI/CD%20Pipeline/badge.svg
 [badge-semver]: https://img.shields.io/github/v/tag/mineiros-io/terraform-aws-iam-role.svg?label=latest&sort=semver
 [badge-license]: https://img.shields.io/badge/license-Apache%202.0-brightgreen.svg
-[badge-terraform]: https://img.shields.io/badge/Terraform-0.13%20and%200.12.20+-623CE4.svg?logo=terraform
+[badge-terraform]: https://img.shields.io/badge/Terraform-0.15%20and%200.12.20+-623CE4.svg?logo=terraform
```

Comment:
13 and 14 should be added to the badge

Reply:
nope, doesn't look appropriate to me - there  is absolutely no value in listing every single version. We should use a semver range instead. Also, a badge shouldn't      block the PR.

Reply:
but the badge is wrong, "0.15 and 0.12.20" is just not true..

Reply:
resolved